src/main/java/seedu/address/model/patient/Age.java

```diff
@@ -11,6 +11,7 @@ public class Age {
     public static final String MESSAGE_CONSTRAINTS =
             "Age should only contain numbers, and it should not be negative";
     public static final String VALIDATION_REGEX = "\\d+";
+    public static String DEFAULT_AGE = "30";
```

Comment:
Hmm, perhaps it will be better for the dummy default values to be more obvious (like '00') or something? So that the user will less likely mix up the entries with non-default values and those with default values (in case they want to come back and fill in the details again). This applies to default birthday and other attributes as well

Reply:
Thanks for pointing that out. Have updated the values to look more default!